pkg/formatters/capabilities.go

```diff
@@ -0,0 +1,86 @@
+package formatters
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/urfave/cli"
+	"github.com/vapor-ware/synse-cli/pkg/scheme"
+	"github.com/vapor-ware/synse-server-grpc/go"
+)
+
+const (
+	// the pretty output format for `server capabilities` requests.
+	prettyServerCapabilities = "{{.Plugin}}\t{{.Kind}}\t{{.Outputs}}\t\n"
```

Comment:
the note made above about the test failing if the golden file didn't have a space at the end could be because of the `\t` at the end here, e.g. instead of `...{{.Outputs}}\t\n`, you could try `...{{.Outputs}}\n`

Reply:
That makes a lot of sense now.